common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -163,13 +163,13 @@ stop_function() {
 }
 
 main_agent_operational() {
-    pgrep -la beerocks_agent | grep -v wlan > /dev/null 2>&1 || return 1
+    pgrep -fx "@BEEROCKS_INSTALL_PATH@/bin/beerocks_agent" > /dev/null 2>&1 || return 1
```

Comment:
The commit message should definitely explain why this change is needed...

Reply:
Updated the commit message